db/schema.rb

```diff
@@ -58,8 +60,10 @@
     t.string   "type"
     t.integer  "parent_id"
     t.integer  "min_children_needed"
+    t.datetime "deleted_at"
   end
 
+  add_index "approvals", ["deleted_at"], name: "index_approvals_on_deleted_at", using: :btree
```

Comment:
ah yes, darn I always forget to roll back migrations when I have two ongoing PRs with migrations. good catch!!